### DISCUSSION TITLE
Station Dropping/Adding protocol

### DISCUSSION THREAD

**Discussion by zoyafuso-NOAA** [afsc-gap-products/StationAllocationAIGOA#9] [General]
@Ned-Laman-NOAA @MargaretSiple-NOAA @Lewis-Barnett-NOAA 

**To drop or to add, that is the question**

**Excerpt from @Ned-Laman-NOAA  (August 25th 2022):**

"Our typical allocation for a 2 boat GOA survey has been 550 and it assumes a 4.3-4.4 station per day average production rate for an average of around 125 fishing days for the summer survey.

If we change the expectation to 4 stations per day and target an allocation based on 125 fishing days, we'd allocate 500 stations to the two boats.

If we do that, here's my thought and question. Let the boats do as many more stations as they'd like to above and beyond the 4 per day on average expectation AND let them pick their stations as they would alternates with some guidelines around how many bonus stations to add to any particular stratum. Since the GOA really progresses from east to west without returning to DH, there are some different considerations in how we go about this for the Gulf vs the Aleutians.

Since this is tied in with the station allocation exercise, and since I believe that Megsie and I have discussed potentially modernizing the station allocation code for the Gulf (currently maintained and run by Paul in SPlus), I've included her on this initial email."

**Excerpt from @MargaretSiple-NOAA (August 26th 2022):**
"
I think this sounds reasonable.  It looks like historically, with an allocation around 550 and using the Station Dropping Tool as needed, the survey completed 500-550 stations, so we can probably expect that the boats will complete something similar and we just need to make sure the policies for picking up additional stations make sense and won't affect the CVs. And maybe have a statistical/philosophical discussion about whether it is better to drop stations or pick them up. My calendar is up to date for September, so schedule away!
"

**Excerpt from @zoyafuso-NOAA (August 29th 2022)**
"
I am fine with starting with an initial allocation of 500 station locations. I envision getting those extra 50 stations from performing a 550 station allocation and then getting the extra 50 stations that allocation. So if stratum X has 10 stations from the original 500 stations and 12 stations from a 550 station allocation, FPCs/skippers can add 2 stations to stratum X. We would then need to discuss whether it makes sense for those two extra stations to be chosen randomly a priori (kind of like what we did in the AI this past summer) or ad hoc by the skippers/FPCs potentially based on proximity to other stations (maybe this is how alts are chosen currently?). I see pros and cons to each approach (potential added bias vs. convenience of doing stations together) and it requires a broader discussion of the statistical/philosophical/practical considerations in adding/dropping stations.
"

**Excerpt from @Ned-Laman-NOAA (August 29th 2022)** 
"
I'm inclined to allocate less and leave room for vessels to excel by adding stations to their allotment. Based on feedback I'm receiving from a variety of people, the psychology of this approach was appreciated more than a psychology of you have 550 stations allotted but didn't finish your allotment. Total aside here but it strikes me as odd that allotted has two t's and allotment has one t!

Anyway, you've both hit the on one of the main topics we need to discuss which is how picking up those "bonus" stations should happen. There may also be some considerations for how we pick up extra stations that are related to the restratification so I'm really looking forward to having this discussion.
"

**Comment by Ned-Laman-NOAA** [comment]
What if picking up "bonus" stations looks a lot like assigning new, previously untrawled stations in the Aleutians. I.e., We assign priority to areas or strata where we'd like to have more stations if time allows. Then skipper/FPC can choose new ground or previously towed ground as suits the time available?

**Reply by zoyafuso-NOAA** [reply]
I think Ned's suggestion will work here where we start with a 500-station allocation but have these 50 bonus stations that FPCs can flexibly choose to do with available time and towable ground. The bonus areas can be picked up like how we did it in the Aleutians in 2022. We assign priority of bonus stations to strata that would require more stations under a 550-station allocation. For example, if stratum 310 has 4 prescribed stations under a 500-station allocation and 6 stations under a 550-station allocation, 2 of the 50 bonus stations would come from any trawlable station in stratum 310.  

I think this rule works regardless of the design we use in 2023.

**Comment by Ned-Laman-NOAA** [comment]
I agree with Zack here in terms of the strategy for our survey allocation this summer. 
In terms of the numbers, I was thinking that last summer we ended up completing the Aleutian survey with a 5% reduction to the historic allocation of 420. My plan is to apply a similar reduction to the historic 2 boat, 550 station allocation for the Gulf so that we would allocate 520 stations and have 30 bonus stations in reserve.
How will we have Paul run the allocation to identify the total 550 stations with 520 regular and 30 bonus stations? or will Zack do that after the fact?
